app/helpers/application_helper.rb

```diff
@@ -597,4 +597,11 @@ def concurrent_events(event)
     end
     concurrent_events
   end
+
+  def speaker_selector_input(form)
+    users = User.select(:id, :name, :email).where(is_disabled: false).order(:name)
```

Comment:
Also, because `User#name` looks up `username` when `name` is `nil`, you must also `select :username`.